sdk/identity/Azure.Identity/src/AzureCliCredential.cs

```diff
@@ -20,6 +20,7 @@ namespace Azure.Identity
     /// </summary>
     public class AzureCliCredential : TokenCredential
     {
+        private readonly AzureCliCredentialOptions _options;
```

Comment:
Again here we should capture the properties out of the options to avoid changing behavior after construction.